homebrew/using_different_homebrew_formula_versions.md

```diff
@@ -0,0 +1,31 @@
+# Using different Homebrew formula versions
+
+Today, I was trying to run the executable from a gem and got the following error:
+
+```bash
+This installation of RMagick was configured with ImageMagick 6.8.9 but
+ImageMagick 6.9.0-3 is in use. (RuntimeError)
+```
```

Comment:
Since this is just an error message now, what are your thoughts on using a quote instead of code block? I'm fine either way. If you do decide to stick to the code block, it should probably not use `bash` syntax highlighting since it incorrectly highlights `in`.

> This installation of RMagick was configured with ImageMagick 6.8.9 but ImageMagick 6.9.0-3 is in use. (RuntimeError)

VS

```
This installation of RMagick was configured with ImageMagick 6.8.9 but
ImageMagick 6.9.0-3 is in use. (RuntimeError)
```